lib/matplotlib/image.py

```diff
@@ -578,9 +578,13 @@ def write_png(self, fname):
 
     def set_data(self, A):
         """
-        Set the image array
+        Set the image array.
 
-        ACCEPTS: numpy/PIL Image A
+        Parameters
+        ----------
+        A : array-like.
+
+        Note that this function does *not* update the normalization used.
```

Comment:
Can you put this above the `Parameters`?  Not sure how this will render with numpydoc.